client/ayon_blender/api/pipeline.py

```diff
@@ -460,25 +466,27 @@ def _discover_gui() -> Optional[Callable]:
     return None
 
 
-def add_to_avalon_container(container: bpy.types.Collection):
-    """Add the container to the Avalon container."""
+def add_to_ayon_container(container: bpy.types.Collection):
+    """Add the container to the Ayon container."""
 
-    avalon_container = bpy.data.collections.get(AVALON_CONTAINERS)
-    if not avalon_container:
-        avalon_container = bpy.data.collections.new(name=AVALON_CONTAINERS)
+    ayon_container = bpy.data.collections.get(AYON_CONTAINERS) or (
+        bpy.data.collections.get("AVALON_CONTAINERS")
+    )
+    if not ayon_container:
+        ayon_container = bpy.data.collections.new(name=AYON_CONTAINERS)
```

Comment:
Technically - this would make it non-backwards compatible in the way that if one wanted to downgrade, these containers wouldn't get picked up. Not sure how strict we need to be there? @philippe-ynput @iLLiCiTiT ?

Reply:
You're right, but I'm afraid that we can't really make it backwards compatible.

NOTE Release with this PR will raise version to `1.0.0`.

Reply:
It is missing feature in the CI, looks like it would require changes in at about 3 CI actions. But the release can be done by manually passing new version in release trigger action.